mesmerize_napari/napari1d_manager.py

```diff
@@ -47,13 +48,22 @@ def __init__(self, batch_item: pd.Series, roi_type: str):
         self.eval_gui = EvalComponentsWidgets(cnmf_viewer=self)
         self.eval_gui.show()
 
-        movie_path = str(batch_item.caiman.get_input_movie_path())
-        if movie_path.endswith('mmap'):
-            Yr, dims, T = load_memmap(movie_path)
-            images = np.reshape(Yr.T, [T] + list(dims), order='F')
-            self.viewer.add_image(images, name="Movie", colormap='gnuplot2')
-        else:
-            self.viewer.open(movie_path, colormap='gnuplot2')
+        self.viz_gui = VizWidget(cnmf_viewer=self.viewer, batch_item=batch_item)
+        self.viewer.window.add_dock_widget(self.viz_gui, area='bottom', add_vertical_stretch=False,name="Visualization")
+        #self.viz_gui.show()
+
+        # Load correlation map first
+        corr_img = batch_item.caiman.get_correlation_image()
+
+        self.viewer.add_image(corr_img, name=f'corr: {batch_item["name"]}', colormap='gray')
```

Comment:
we can use the config module so the user can set whatever default cmap they want https://github.com/nel-lab/mesmerize-napari/pull/28